docs/en/enterprise-edition/content-collections/dashboards/create-and-manage-dashboards.adoc

```diff
@@ -322,6 +323,12 @@ Share the custom dashboards you have created with members of your organization o
 .. We recommend that you guide users to toggle on the the *Visibility* setting of the shared Dashboard under *Dashboards> Menu > Manage Dashboards*. 
 .. Once the dashboard is shared globally, it will appear by default at the bottom of the *Manage Dashboards* list and can be reordered to the top by individual users for easy access. If the user you wish to share a dashboard with, is currently on the Dashboards page, they will also need to refresh the page to see the shared dashboard. 
 
-NOTE: If you are a user and you've received a URL for a shared Dashboard and you haven't updated the dashboard's *Visibility* settings, the shared Dashboard will only show up temporarily on the Menu tab. To ensure it's always accessible follow the steps listed above. 
+[NOTE] 
+====
+Keep the following caveats in mind when sharing dashboards: +
 
+* If you are a user and you've received a URL for a shared Dashboard and you haven't updated the dashboard's *Visibility* settings, the shared Dashboard will only show up temporarily on the Menu tab. To ensure it’s always accessible from the Menu tab follow the steps listed above. +
```

Comment:
If you received a shared Dashboard URL and haven’t updated the dashboard’s *Visibility* settings, the shared Dashboard will only show up temporarily on the Menu tab. To ensure it’s always accessible from the Menu tab, follow the steps listed above.